### ISSUE TITLE
MessageDialog not inheriting dynamically assigned accent

### ISSUE THREAD

**Issue by TayebJa3ba** [MahApps/MahApps.Metro#2182]
Hi, 
When I use `MetroWindow.ShowMessageAsync` with a statically loaded ressource dict (red for example) it works great and inherits the accent from parent window. When loading it programatically, e.g parsing it from settings or giving user choice to change accents, it gives this all the time (note the different accents: blue in the dialog, and green in the form)

![untitled](https://cloud.githubusercontent.com/assets/11261296/10742510/e29e567e-7c2d-11e5-8878-5a72196426ff.jpg)

Is it a bug, or me missing something ? 

Thank you !

**Comment by TayebJa3ba** [comment]
I've figured it out on my own, found a workaround. I had to call ChangeAppStyle like this: 
`ThemeManager.ChangeAppStyle(Application.Current.Resources, ThemeManager.GetAccent(_col), ThemeManager.GetAppTheme(color));`
            
It didn't work, so I had to add the accent and the theme to the app.xaml.
`
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/Cobalt.xaml" />
<ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/BaseLight.xaml" />
`

There is an another issue, when none of those dicts is statically loaded, DetectAppStyle can return null, and cause bugs later. I'll try to make a patch later. 

Thanks for the superb library anyway !
